orion/sites/admin/src/achieve/achieve.test.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import { AchievePage } from "./achieve.js";
+
+describe("test", () => {
+    const component = shallow(<AchievePage />);
+
+    test("renders", () => {
+        expect(component.exists()).toBe(true);
+        expect(component.find("h1").text()).toContain("All Achievements");
+        expect(component.find("Link").text()).toContain("Add Achievement");
+        expect(component.find("AchieveRow").length).toBe(0);
+    });
```

Comment:
The above is the "main" test we really want to make sure we have for every Component. This ensures that the component will always render. If there's a typo or a syntax anywhere, the component will not render and thus this test will fail.